diff --git a/node_modules/react-native-draganddrop-board/src/components/Column/Column.js b/node_modules/react-native-draganddrop-board/src/components/Column/Column.js
index 97c232e..cf86608 100644
--- a/node_modules/react-native-draganddrop-board/src/components/Column/Column.js
+++ b/node_modules/react-native-draganddrop-board/src/components/Column/Column.js
@@ -1,5 +1,5 @@
 import React from 'react'
-import { FlatList } from 'react-native'
+import { FlatList,Text,TouchableOpacity,View } from 'react-native'
 import {
   bool,
   func,
@@ -23,7 +23,9 @@ import {
   SumWrapper
 } from './Column.styled'
 
-const COLUMN_WIDTH = 0.78 * deviceWidth
+
+
+const COLUMN_WIDTH = 0.65 * deviceWidth
 const PADDING = 32
 const ONE_COLUMN_WIDTH = deviceWidth - PADDING
 
@@ -80,6 +82,7 @@ class Column extends React.Component {
       item
     }
     return (
+    
       <RowWrapper
         ref={ref => this.setItemRef(item, ref)}
         collapsable={false}
@@ -88,6 +91,7 @@ class Column extends React.Component {
       >
         {renderWrapperRow(props)}
       </RowWrapper>
+      
     )
   }
 
@@ -193,6 +197,7 @@ class Column extends React.Component {
         width={oneColumn ? ONE_COLUMN_WIDTH : COLUMN_WIDTH}
         marginRight={oneColumn ? 0 : 8}
       >
+       
         <RowContainer>
           <Paragraph
             fontSize={columnNameFontSize}
@@ -201,7 +206,9 @@ class Column extends React.Component {
           >
             {column.data().name}
           </Paragraph>
-          {isWithCountBadge && <SumWrapper>
+          {isWithCountBadge && 
+          <SumWrapper>
+            
             <ParagraphWrapper
               backgroundColor={badgeBackgroundColor}
               width={badgeWidth}
@@ -215,11 +222,25 @@ class Column extends React.Component {
                 lineHeight={ios ? null : badgeTextFontSize * 1.6}
               >
                 {colElements.toString()}
+                
               </Paragraph>
+              
             </ParagraphWrapper>
+            
+            <TouchableOpacity onPress={this.props.burgerClick}>
+          <View>
+            <View style={{backgroundColor:'grey',height:4,width:4,borderRadius:4}}/>
+            <View style={{backgroundColor:'grey',height:4,width:4,borderRadius:4}}/>
+            <View style={{backgroundColor:'grey',height:4,width:4,borderRadius:4}}/>
+          </View>
+          </TouchableOpacity>
           </SumWrapper>
+          
           }
+          
+
         </RowContainer>
+        
         {boardRepository
           .items(column.id()).length - 1 === 0 ?
           (emptyComponent
@@ -242,6 +263,11 @@ class Column extends React.Component {
             enableEmptySections
           />
         }
+        <View style={{justifyContent:'center',alignItems:'center',marginBottom:10}}>
+        <TouchableOpacity onPress={() => this.props.AddNewCard(column.data().name)}>
+          <Text style={{fontSize:14}}>+ Add New Card</Text>
+        </TouchableOpacity>
+        </View>
       </ColumnWrapper>
     )
 
diff --git a/node_modules/react-native-draganddrop-board/src/components/Column/Column.styled.js b/node_modules/react-native-draganddrop-board/src/components/Column/Column.styled.js
index fc0f198..613f2ba 100644
--- a/node_modules/react-native-draganddrop-board/src/components/Column/Column.styled.js
+++ b/node_modules/react-native-draganddrop-board/src/components/Column/Column.styled.js
@@ -18,10 +18,12 @@ const ColumnWrapper = styled.View`
 const ParagraphWrapper = styled.View`
   alignItems: center;
   justifyContent: center;
+  marginRight:15;
 `
 
 const RowContainer = styled.View`
   flexDirection: row;
+  justifyContent: space-between;
   alignItems: center;
   paddingVertical: 18;
   paddingHorizontal: 10;
@@ -36,12 +38,15 @@ const Paragraph = styled.Text`
 
 const RowWrapper = styled.View`
   opacity: 1;
+  alignItems: center;
+  justifyContent: center;
 `
 
 const SumWrapper = styled.View`
   marginLeft: 8;
   alignItems: center;
   justifyContent: center;
+  flexDirection: row;
 `
 
 export {
diff --git a/node_modules/react-native-draganddrop-board/src/constants/colors.js b/node_modules/react-native-draganddrop-board/src/constants/colors.js
index 0a9ccd6..f9e53ac 100644
--- a/node_modules/react-native-draganddrop-board/src/constants/colors.js
+++ b/node_modules/react-native-draganddrop-board/src/constants/colors.js
@@ -2,7 +2,7 @@ const colors = {
   bay: '#747d8c',
   black: '#000000',
   blurple: '#4834d4',
-  deepComaru: '#30336b',
+  deepComaru: 'white',
   exodusFruit: '#686de0',
   fallingStar: '#FAFAFA',
   officer: '#2C3A47',
